test/zipkin-simple.test.js

```diff
@@ -168,39 +150,52 @@ describe("Zipkin client", function () {
 					done()
 				})
 
-				Client.clientSend(traceData, {
+				Client.sendClientSend(traceData, {
```

Comment:
I agree it's a bit wordy, but I preferred to be more descriptive here.
I added the send prefix to avoid confusion especially on what the methods were doing (i.e. clientRecv could give the idea of waiting for something to be received).

I kinda like startSpan, but I want to keep it coherent with the other methods.

Another good options could be to have startSpan, startChildSpan (which would make getChild obsolete), and finishSpan - but I can't come up with something similar for the server annotations